.idea/workspace.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think we need this file `.idea/workspace.xml` to be merged into master. 🤔

Reply:
Yes, that wasn't in my .gitignore so got commited accidentally :sweat_smile: . Removed it on my latest commit .